pkgs/rust-internal-deps/default.nix

```diff
@@ -71,6 +74,7 @@ in
           ++ [
             gitMinimal
             process-compose
+            coreutils # for `sleep infinity`
```

Comment:
Note to self: this builds upon the fix for `git` not being included in the closure in https://github.com/flox/flox/pull/1926